docs/source/notes/multiprocessing.rst

```diff
@@ -20,22 +20,23 @@ memory and will only send a handle to another process.
 This allows to implement various training methods, like Hogwild, A3C, or any
 others that require asynchronous operation.
 
-Sharing CUDA tensors
---------------------
-
-Sharing CUDA tensors between processes is supported only in Python 3, using
-a ``spawn`` or ``forkserver`` start methods. :mod:`python:multiprocessing` in
-Python 2 can only create subprocesses using ``fork``, and it's not supported
-by the CUDA runtime.
+CUDA in multiprocessing
+-----------------------
 
-.. warning::
+The CUDA runtime does not support the ``fork`` start method. However,
+:mod:`python:multiprocessing` in Python 2 can only create subprocesses using
+``fork``. So Python 3 and either ``spawn`` or ``forkserver`` start method are
+required to use CUDA in subprocesses.
 
-    CUDA API requires that the allocation exported to other processes remains
-    valid as long as it's used by them. You should be careful and ensure that
-    CUDA tensors you shared don't go out of scope as long as it's necessary.
-    This shouldn't be a problem for sharing model parameters, but passing other
-    kinds of data should be done with care. Note that this restriction doesn't
-    apply to shared CPU memory.
+.. note::
+  The start method can be set via either creating a context with
+  ``multiprocessing.get_context(...)`` or directly using
+  ``multiprocessing.set_start_method(...)``.
+
+Unlike CPU tensors, the sending process is required to keep the original tensor
+as long as the receiving process retains a copy of the tensor. It is implemented
+under the hood but requires users to follow the best practices described in
+:ref:`this section <multiprocessing-cuda-sharing-details>`.
```

Comment:
We should still warn that even this refcounting can't save you if the child process exits!

Reply:
Sure I will add a sentence. Do you mean that it can't save you in the sense that
1. the refcount is not decremented when child process exits abnormally, 
or 
2. the child process will segfault if the sending process exits?

Reply:
I added both. Let me know if they aren't what you expected, or are incorrect.